cleo/io/inputs/token_parser.py

```diff
@@ -31,7 +31,7 @@ def _parse(self) -> list[str]:
         tokens = []
 
         while self._is_valid():
-            if self._current.isspace():
+            if self._current is not None and self._current.isspace():
```

Comment:
`mypy` does not consider `while self._is_valid()` as a sufficient type check. I might revisit this in refactoring later.